src/web/webpack.config.js

```diff
@@ -34,6 +46,18 @@ module.exports = {
           name: '[name].[ext]',
         },
       },
+      {
+        test: /\.tsx?$/,
+        use: [
+          {
+            loader: require.resolve('ts-loader'),
```

Comment:
What `tsconfig.json` we use here?
If `src/tsconfig.json` then we could cut `src/bundle/tsconfig.json`.